cmd/createcptv/main.go

```diff
@@ -20,36 +20,49 @@ import (
 	"time"
 
 	"github.com/TheCacophonyProject/go-cptv"
-	"github.com/TheCacophonyProject/lepton3"
+	"github.com/TheCacophonyProject/go-cptv/pkg/cptvframe"
```

Comment:
why have `/pkg` in there? Why not just `github.com/TheCacophonyProject/go-cptv/cptvframe`?

Reply:
should cptvframe directory just live under go-cptv then?

Reply:
Given that lepton and boson packages might be importing it and only need what's in there then perhaps keep it in it's own package